lib/packwerk/cli.rb

```diff
@@ -87,11 +87,13 @@ def init
     end
 
     def generate_configs
+      checker.validate_rails_app!
```

Comment:
we actually don't need the app to be a rails app anymore in order to generate configs - right?

Reply:
No I suppose not -- removed it